agent/installer/cli.go

```diff
@@ -6,19 +6,20 @@ package installer
 import (
 	"flag"
 	"fmt"
-	"k8s.io/klog"
+	"github.com/go-logr/logr"
 	"k8s.io/klog/klogr"
 )
 
 var (
-	listFlag             = flag.Bool("list", false, "List all supported OS and Kubernetes versions")
+	listSupportedFlag    = flag.Bool("listSupported", false, "List all supported OS and Kubernetes versions")
 	listBundlesFlag      = flag.Bool("listBundles", false, "List the BYOH Bundle names for all supported OS and Kubernetes versions")
 	detectOSFlag         = flag.Bool("detect", false, "Detects the current operating system")
 	installFlag          = flag.Bool("install", false, "Install a BYOH Bundle")
 	uninstallFlag        = flag.Bool("uninstall", false, "Unnstall a BYOH Bundle")
-	bundleRepoFlag       = flag.String("bundleRepo", "https://projects.registry.vmware.com", "BYOH Bundle Repository. If not set, will look for bundles locally")
+	bundleRepoFlag       = flag.String("bundleRepo", "projects.registry.vmware.com", "BYOH Bundle Repository")
+	cachePathFlag        = flag.String("cachePath", ".", "Path to the local bundle cache")
 	k8sFlag              = flag.String("k8s", "1.22.1", "Kubernetes version")
-	osFlag               = flag.String("os", "", "OS. If used with install/uninstall, skip os detection")
+	osFlag               = flag.String("os", "", "OS. If used with install/uninstall, override os detection")
```

Comment:
Question: Is it better to have 2 words args names with lower case and a dash, e.g. `--list-supported`, `--list-bundles`. I am not sure if there is some convention for args naming but I have seen some CLIs having lowercase and dash.